appstudio-utils/util-scripts/verify-attestation-with-policy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Verifies the given attestation passes the given rego policy
+# usage:
+#   verify-attestation-with-policy.sh <attestation file> <output file> <passed file>
+# where:
+#   <attestation file> file containing the attestation to be verified
+#   <output file>      where to store the result in JSON format
+#   <passed file>      where to store the overall passing result
+set -euo pipefail
+
+attestation="$1"
+output="$2"
+passed="$3"
+
+predicate=$(jq '.payload | @base64d | fromjson' ${attestation})
+
+cd $(dirname $0)
+source lib/fetch.sh
+
+# TODO: If git clone fails, the task does not fail - fix that!
+./fetch-ec-policies.sh
+
+# TODO: fix the checks (other than not_useful) to handle the case when there
+# is no cluster access, i.e. no TaskRun yaml
+# When fixed, replace this line with a call to save-policy-config
+echo '{"non_blocking_checks":[
+    "not_useful",
+    "chains_config",
+    "cluster_sanity",
+    "transparency_urls",
+    "transparency_log_attestations"
+    ]}' | jq > $( json-data-file config policy )
```

Comment:
Probably won't need this after https://github.com/hacbs-contract/ec-policies/pull/11 is merged, which might happen in a minute or two...

Reply:
Though.. the rego rules in there do make no incorrect assumptions about the location of attestations in the $EC_WORK/data directory.

Reply:
True. I can play around with that repo as a way to get this PR further along.

Reply:
With that PR, this is no longer needed. Removed it. (I'll still follow up on adjusting ec-policies further)